Fixes #1334:  visited link should not change color on homepage

rishavpandey43 opened the pull request:
<!-- Add issue number here. If you do not solve the issue entirely, please change the message e.g. "Addresses #IssueNumber -->
Fixes #1334 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link
<!-- Replace "PR_NUMBER" with your pull request number. This link is generated when your PR passes the travis tests.A sample link can look like https://pr-200-fossasia-susper.surge.sh -->

#### Changes proposed in this pull request:

<!-- Changes: Add here what changes were made in this issue and if possible provide links. -->
- I've fixed the issue of changing of color of link visited in homepage

<!-- Demo Link: Add here the link where you changes can be seen. -->

- https://pr-1338-fossasia-susper.surge.sh

<!-- Screenshots for the change: Add here the screenshot of the fix. -->
-

rishavpandey43 commented:
@shaharyar-shamshi 
changed the code from style.css to css/base.css

rishavpandey43 commented:
@shaharyar-shamshi 
Thanks a lot for your review, since I'm new in the open source contribution, I faced some issue to organise the code in proper way.
Hope to see your help in coming future, if required.

shreyanshdwivedi commented:
@rishavpandey43 I think u wanted to tag me :sweat_smile: 
I'll be there to help. Btw please squash your commits

rajvaibhavdubey commented on src/.fuse_hidden0000271c00000002:
```diff
@@ -0,0 +1 @@
+/* You can add global styles to this file, and also import other style files */
```
I don't think that this file is needed.

rishavpandey43 replied:
This file was added by default, I've even removed it.

rishavpandey43 commented:
@shreyanshdwivedi @rajvaibhavdubey 
Thank you'll for review, I've removed the extra file and also squashed the commits.
Please, have a look once again and suggest if any change is needed more.

mariobehling commented:
Closing due to inactivity.
